src/hooks/useNombrePersona.js

```diff
@@ -0,0 +1,16 @@
+import {useState} from "react";
+
+const useNombrePersona = (initialState) => {
+    const [nombre, setStateNombre] = useState(initialState);
+    const [caracteres, setStateLength] = useState(0);
```

Comment:
Nice!! but instead of the name **caracteres**, I would prefer a name more in context line lengthofcaracteres,, but it is ok